docker-compose.override.yml

```diff
@@ -39,6 +39,20 @@ services:
           restart: on-failure
           links:
               - data-api
+    ganache:
+        container_name: streamr_dev_ganache
+        image: streamr/streamr-ganache:dev
+        ports:
+          - "8545:8545"
+        restart: on-failure
+        environment:
+            EE_URL: http://engine-and-editor:8081/streamr-core
```

Comment:
The URL given here won't work without the `links` section which was removed. This line leads to believe that the ganache container accesses `engine-and-editor` for some reason?

Reply:
it does, for getting test products that are added to Marketplace contract